src/app/article/article.component.css

```diff
@@ -0,0 +1,11 @@
+.article-read-section{
+    border: solid black 5px;
```

Comment:
Same change as mentioned for `article-list.component.css` file

Reply:
**Changes Implemented**